docker/script/entrypoint.sh

```diff
@@ -41,10 +41,11 @@ package_requirements() {
     if [[ -e "$AIRFLOW_HOME/$REQUIREMENTS_FILE" ]]; then
         echo "Packaging requirements.txt into plugins"
         pip3 download -r "$AIRFLOW_HOME/$REQUIREMENTS_FILE" -d "$AIRFLOW_HOME/plugins"
+        wget "https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION%.*}.txt" -O $AIRFLOW_HOME/plugins/constraints.txt
```

Comment:
If you compare the local runner images' constraints file against the one published by Apache Airflow, you will notice there are a few differences for some versions. The source of truth should be the constraints.txt file which exists in this repository. Additionally; users can make changes to this constraints file to fit their use-cases. You can copy it into the plugins directory.

I am also wondering if it's better to place the constraints file in the plugins directory in the `package-requirements()` function instead of being a step in the startup script. This will ensure that the constraint file will be populated if the user runs the command multiple times, and deletes the directory files in between them.

Reply:
Modified to fetch the constraints from `aws-mwaa-local-runner` instead of airflow.
Also for consistency, would it be wise to also switch out the constraint file in `requirements/requirements.txt` since it is also currently pointing to the apache airflow constraint instead of this repo?
`--constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.7.2/constraints-3.11.txt"`

On your last point, do you mean to populate the constraint in that startup script instead of this `package-requirements` function?

I